pkg/defaultcomponents/defaults.go

```diff
@@ -141,18 +163,28 @@ func Components() (otelcol.Factories, error) {
 		prometheusexporter.NewFactory(),
 		fileexporter.NewFactory(),
 		kafkaexporter.NewFactory(),
-		sapmexporter.NewFactory(),
-		signalfxexporter.NewFactory(),
-		datadogexporter.NewFactory(),
-		logzioexporter.NewFactory(),
 		loggingexporter.NewFactory(),
 		otlpexporter.NewFactory(),
 		otlphttpexporter.NewFactory(),
 		awsxrayexporter.NewFactory(),
 		loadbalancingexporter.NewFactory(),
 		awscloudwatchlogsexporter.NewFactory(),
 	}
-
+	if datadogExporterFeatureGate.IsEnabled() {
```

Comment:
See above about naming.  This condition should be inverted:

```suggestion
	if !datadogExporterFeatureGate.IsEnabled() {
```

so that in the alpha stage where the gate is disabled by default the component is available.

Reply:
I was initially trying to have component by default and hense started from reverse fashion, like from beta, then alpha -> removal. instead the actual should alpha -> deprecated -> removal. 

It makes sense. Understood the idea now.